### DISCUSSION TITLE
In the usage example why are there two enums (my_state, my_status)?

### DISCUSSION THREAD

**Discussion by amitjindal** [Borderless360/django-logic#67] [Q&A]
Hi, I am trying to understand this. Why do we need two states in Invoice model?

The name: **MY_STATE_CHOICES** seems to indicate that the **my_state** is the right enum field for model and you are in fact binding to the same.

But the default specified is wrong in that. **Default='open'** is not a valid choice in MY_STATE_CHOICES.
Whereas the **my_status** specifies correct default but it does not seems to be used.

So my question is, if I want to use the package, do I need to specify two states in every model? How will the current sample work since there is no transition that provides source as 'open'.

Please clarify.

**Comment by edigerad** [comment]
Thank you for pointing this out! It looks like `my_status` is unnecessary and was left by mistake. The correct field is `my_state`, and its default value should match a valid choice in `MY_STATE_CHOICES`.  You don't need to define two states in your model—just use `my_state`. Let me know if you encounter any further issues!
